crates/pipeline_manager/src/api/auth_api.rs

```diff
@@ -8,7 +8,7 @@ use super::{ManagerError, ServerState};
 
 /// Get authentication provider configuration
 #[utoipa::path(
-    path="/config/authentication",
+    path="/../config/authentication",
```

Comment:
Why is this change required? This was fixed to `/config/authentication` as part of https://github.com/feldera/feldera/issues/1084.

Reply:
Otherwise TypeScript generates a url /v0/config/authentication for this action, which is wrong. I understand this path is relative (a subpath), hence the issue

Reply:
Ah, now I see why you need this.

Our use of a `Servers` variable is the issue: https://github.com/feldera/feldera/blob/main/crates/pipeline_manager/src/api/mod.rs#L63

I'll see if I can find a quick workaround so you don't need this change. Basically, we need the `/v0` prefix only for the authenticated APIs.

Reply:
@Karakatiza666 btw, I've been using the authentication code from the UI just fine. What workflow breaks without the relative pathing?

Reply:
You are using non-updated UI code. I re-generated UI OpenAPI bindings and there this issue popped up, so I had to add '/../' to get generated code to behave as before.
I think I added this previously too, actually.

Reply:
Ack. I've opened https://github.com/feldera/feldera/pull/1139. See if that fixes your problem. If so, let's merge that and rebase this PR on top of it.